microsoft-edge/webview2/release-notes/index.md

```diff
@@ -67,11 +67,344 @@ See also:
 ###### SDK-only
 -->
 
-<!-- end of Jul 2024 Release SDK -->
-<!-- end of Jul 2024 Prerelease SDK -->
+<!-- end of Sep 2024 Release SDK -->
+<!-- end of Sep 2024 Prerelease SDK -->
 <!-- end of template -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2739.15
+
+Release Date: August 26, 2024
+
+[NuGet package for WebView2 SDK 1.0.2739.15](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2739.15)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 128.0.2739.15 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+The following APIs have been promoted to Stable and are now included in this Release SDK.
+
+
+<!-- ------------------------------ -->
+* Added `Notification` APIs that support web notifications, including non-persistent notifications and persistent notifications.  A _non-persistent notification_ is a notification without an associated service worker registration.  A _persistent notification_ is a notification with an associated service worker registration.  You should be able to handle<!-- todo: "Your app should handle"?  "Your app can optionally handle"? --> notification permission requests, and further listen to<!-- todo: "and then listen for"? --> `Notification Received` events to optionally handle the notifications themselves.<!-- todo: delete "themselves"?  does "should" contradict "optionally"? -->
```

Comment:
The notification APIs only cover the non-persistent notifications but not persistent notifications. The summary should be the same/similar to the description in 2730-prerelease.

Reply:
result:
```
* Added support for Web Notification, for non-persistent notifications.  The `NotificationReceived` event for `CoreWebView2` controls web notification handling, allowing customization or suppression by the host app.  Unhandled notifications default to WebView2's UI.
```